include/swift/ABI/MetadataValues.h

```diff
@@ -2485,6 +2486,9 @@ class JobFlags : public FlagSet<uint32_t> {
   FLAGSET_DEFINE_FLAG_ACCESSORS(Task_IsAsyncLetTask,
                                 task_isAsyncLetTask,
                                 task_setIsAsyncLetTask)
+  FLAGSET_DEFINE_FLAG_ACCESSORS(Task_HasInheritedExecutorPreference,
+                                task_hasInitialTaskExecutorPreferenceRecord,
+                                task_setHasInitialTaskExecutorPreferenceRecord)
```

Comment:
Two different names for the same concept

Reply:
So the "initial" is slightly different. It is one we create from the "on:" parameters and we need to know if we did so in order to destroy it properly as we destroy the task in "complete". Other preference records just are destroyed using an adequate "pop" pair to the "push" calls.

Reply:
Oh whoops i see Inherited/Initial, yes it should be Initial -- thank you